docs/intro.asciidoc

```diff
@@ -17,8 +17,9 @@ as well as a simple API which allows you to instrument any application.
 [[additional-components]]
 === Additional Components
 
-APM Agents work in conjunction with the {apm-server-ref}/index.html[APM Server], {ref}/index.html[Elasticsearch], and {kibana-ref}/index.html[Kibana].
-Please view the {apm-get-started-ref}/index.html[APM Overview] for details on how these components work together. 
+APM Agents work in conjunction with the {apm-server-ref-v}/index.html[APM Server],
+{ref}/index.html[Elasticsearch], and {kibana-ref}/index.html[Kibana].
```

Comment:
A little bit inconsistent line break. Either the entire sentence should be on the same line like in the old version or there should be a line break after each comma:

```suggestion
{ref}/index.html[Elasticsearch],
and {kibana-ref}/index.html[Kibana].
```